docs/Rootless.md

```diff
@@ -33,19 +35,32 @@ endif
 ```
 
 - The iOS 14 arm64e ABI mentioned in [arm64e Deployment](arm64e-Deployment.html) is now *required* for the relevant devices
-    - Currently, it's only possible to build for the new ABI on macOS
-        - If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
+    - Currently, it's only possible to build for the new ABI on macOS (as only the Xcode tools can build for it)
+        - Unfortunately, new toolchain does not support building for the old ABI. If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
     - That being said, developers on other platforms can circumvent this *if necessary*:
-        - By adding `oldabi` as a dependency to their package (preferably only as a last resort)
+        - By [using GitHub Actions to compile your tweaks](https://github.com/p0358/SilentScreenshots/blob/master/.github/workflows/build.yml) and upload them as downloadable artifacts (free for both public and private repos)
+        - By using a macOS virtual machine
+            - On Linux you can use [OSX-KVM](https://github.com/kholia/OSX-KVM) script set
+            - Without working App Store, download full Xcode xip archive and extract it, afterwards move `Xcode.app` to `Applications` folder (you can do this graphically through Finder)
+            - If you want to cut on resources used by the VM:
+                - [Enable SSH access in System Preferences](https://osxdaily.com/2022/07/08/turn-on-ssh-mac/)
+                - Disable the WindowServer daemon with `sudo launchctl disable system/com.apple.WindowServer` and reboot to drop the idle CPU usage and RAM down to ~900 MB (this disables the entire graphical interface of macOS)
```

Comment:
```suggestion
                - Disable the WindowServer daemon with `sudo launchctl disable system/com.apple.WindowServer` and reboot. This will disable macOS's graphical user interface, reducing the idle CPU and RAM usage to ~900 MB
```